api/cues_api.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import division
+from __future__ import print_function
+
+import logging
+
+import common
+import models
+
+# We only accept known cue name strings.
+ALLOWED_CUES = ['progress-checkmarks']
+
+
+class CuesAPI(common.APIHandler):
```

Comment:
It's not clear what this API does in practice.  I'd recommend adding a comment here.

Reply:
Example:  It's unclear why always returning an empty map would be appropriate for non-400 cases.

Reply:
I added docstrings.

It was returning an empty dict simply because that call did not need any return value.  I changed it to send {'message': 'Done'} and added a comment.